tests/test_tasks.py

```
@@ -0,0 +1,14 @@
+# tests/test_tasks.py
+
+def success_task(x, y):
+    """A simple task that succeeds."""
+    return x + y
+
+def failure_task():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)